qa/workunits/rbd/nvmeof_fio_test.sh

```diff
@@ -3,11 +3,46 @@
 sudo yum -y install fio
 sudo yum -y install sysstat
 
+namespace_range_start=
+namespace_range_end=
+rbd_iostat=false
+
+while [[ $# -gt 0 ]]; do
+    case "$1" in
+        --start_ns)
+            namespace_range_start=$2
+            shift 2
+            ;;
+        --end_ns)
+            namespace_range_end=$2
+            shift 2
+            ;;
+        --rbd_iostat)
+            rbd_iostat=true
+            shift
+            ;;
+        *)
+            exit 100	# Internal error
+            ;;
+    esac
+done
+
 fio_file=$(mktemp -t nvmeof-fio-XXXX)
-drives_list=$(sudo nvme list --output-format=json | jq -r '.Devices | .[] | select(.ModelNumber == "SPDK bdev Controller") | .DevicePath')
+all_drives_list=$(sudo nvme list --output-format=json | 
+    jq -r '.Devices | sort_by(.NameSpace) | .[] | select(.ModelNumber == "SPDK bdev Controller") | .DevicePath')
+
+# When the script is passed --start and --end (example: `nvmeof_fio_test.sh --start 1 --end 3`), 
```

Comment:
Why did `--start_ns` change to `--start` in the latest version here (and same for the complement)?  The options themselves are still named `--start_ns` and `--end_ns`.